{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2020-4511",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/182366",
                    "name": "ibm-qradar-cve20204511-dos (182366)",
                    "refsource": "XF",
                    "tags": []
                },
                {
                    "url": "https://www.ibm.com/support/pages/node/6246135",
                    "name": "https://www.ibm.com/support/pages/node/6246135",
                    "refsource": "CONFIRM",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "IBM QRadar SIEM 7.3 and 7.4 could allow an authenticated user to cause a denial of service of the qflow process by sending a malformed sflow command. IBM X-Force ID: 182366."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": {},
    "publishedDate": "2020-07-14T13:15Z",
    "lastModifiedDate": "2020-07-14T14:15Z"
}